components/Contact/ContactForm.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+import React from 'react';
+import { useRouter } from 'next/navigation';
+import { ToastContainer, toast } from 'react-toastify';
+
+import 'react-toastify/dist/ReactToastify.css';
+
+import { sendEmail } from '@/lib/utils';
+import { useTheme } from '@/context/ThemeProvider';
+
+import ToastMessage from './ToastMessage';
+
+const ContactForm = () => {
+  const { mode } = useTheme();
+  const router = useRouter();
+  return (
+    <form
+      action={async (formData) => {
+        const { data, error } = await sendEmail(formData);
+        if (error) {
+          toast.error(
+            'Error sending message, please try again later'
+          );
+        }
+        if (data) {
+          toast.success(
+            ({ closeToast }) => <ToastMessage formData={formData} />,
+            {
+              theme: mode,
+            }
+          );
```

Comment:
```suggestion
          toast.success(
            () => <ToastMessage formData={formData} />,
            {
              theme: mode,
            }
          );
```

No need to declare the variable if we're not using it.